packages/payment-processor/src/payment/swap-any-to-erc20.ts

```diff
@@ -79,9 +79,11 @@ export function encodeSwapToPayAnyToErc20Request(
 
   // check if conversion currency is accepted
   if (
-    !request.extensions[
-      PaymentTypes.PAYMENT_NETWORK_ID.ANY_TO_ERC20_PROXY
-    ].values.acceptedTokens.includes(conversionSettings.currency.value)
+    !request.extensions[PaymentTypes.PAYMENT_NETWORK_ID.ANY_TO_ERC20_PROXY].values.acceptedTokens
+      .map((t: string) => {
+        return t.toLowerCase();
+      })
```

Comment:
```suggestion
      .map(t => t.toLowerCase())
```
We like shortcuts :)